dotnet/message-service/Startup.cs

```diff
@@ -10,9 +10,10 @@
 
 namespace message_service
 {
+    using StackExchange.Redis;

```

Comment:
This should probably be moved to the top, above the namespace. It's pretty rare to scope `using`s inside namespaces.